### ISSUE TITLE
Crash with message about stack guard being disabled wrt libjwgl.so.

### ISSUE THREAD

**Issue by sustained** [MultiMC/MultiMC5#2693]
System Information
-----------------------------
MultiMC version: 0.6.5-1217 lin64 (build 1217)

Operating System: Fedora 29, x86_64

Summary of the issue or suggestion:
----------------------------------------------

Minecraft crashes with this error:

```
[13:25:02] [main/INFO] [FML]:    Patching method func_70617_f_()Z
OpenJDK 64-Bit Server VM warning: You have loaded library .../MultiMC/instances/FTB Presents Skyfactory 3/natives/liblwjgl.so which might have disabled stack guard.
The VM will try to fix the stack guard now. It's highly recommended that you fix the library with 'execstack -c <libfile>', or link it with '-z noexecstack'.
```

It's worth noting that the normal Minecraft launcher works fine with this version of Java.

What should happen:
------------------------------

Shouldn't crash?

Steps to reproduce the issue (Add more if needed):
-------------------------------------------------------------

Attempting to run any mod pack, seemingly. Let me know if you need any information.

Suspected cause:
---------------------------

I guess something to do with the Java version but I tried 4 different ones and as I mentioned before, this version of Java works fine with the normal Minecraft launcher.

Logs/Screenshots:
----------------------------
[//]: # (Please refer to https://github.com/MultiMC/MultiMC5/wiki/Log-Upload for instructions on how to attach your logs.)

https://paste.ee/p/ckiAB

Additional Info:
---------------------------

Java version: openjdk version "1.8.0_212"
OpenJDK Runtime Environment (build 1.8.0_212-b04)
OpenJDK 64-Bit Server VM (build 25.212-b04, mixed mode)

Also attempted with the detected 11.0.3 and 12.0.1.

**Comment by Janrupf** [comment]
Its not really the message that leads to the launch problem. This is the important part:
`Caused by: java.lang.UnsatisfiedLinkError: /home/sustained/Binaries/MultiMC/instances/FTB Presents Skyfactory 3/natives/liblwjgl64.so: libjawt.so: cannot open shared object file: No such file or directory`
Your system is missing the native awt library. You might have installed a headless jdk, you night one with graphics support.

**Comment by sustained** [comment]
Hmm okay but wouldn't the normal Minecraft also need a JDK with graphics support? At least, I would have thought so. So I figured it couldn't be that.

Well, fixed with `sudo dnf install java-1.8.0-openjdk` and by selecting `/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.212.b04-0.fc29.x86_64/jre/bin/java` anyway. In case anyone else finds this via Google or some such.

Sorry for wasting your time and thank you.
